solr/solr-ref-guide/src/analytics.adoc

```diff
@@ -33,7 +33,7 @@ Since the Analytics framework is a _search component_, it must be declared as su
 For distributed analytics requests over cloud collections, the component uses the `AnalyticsHandler` strictly for inter-shard communication.
 The Analytics Handler should not be used by users to submit analytics requests.
 
-To use the Analytics Component, the first step is to install this contrib module's plugins into Solr -- see the <<solr-plugins.adoc#installing-plugins,Solr Plugins>> section on how to do this.
+To use the Analytics Component, the first step is to install this contrib module's plugins into Solr -- see the <<solr-plugins.adoc#installing-plugins,Solr Plugins>> section on how to do this. Note: method with `<lib>` directories doesn't work, also consider copying `solr-analytic-x.x.jar` to `${solr.install.dir}/server/solr-webapp/webapp/WEB-INF/lib/`. 
```

Comment:
The way you worded this, it suggests it might be deployed to two places.  It should be one or the other.  The word "also" and "copy" both contributed to this misunderstanding.

Reply:
Agreed. I would phrase it as:

> Note: Method with `<lib>` directive doesn't work. Instead copy `solr-analytic-x.x.jar` to `${solr.install.dir}/server/solr-webapp/webapp/WEB-INF/lib/`, as described in the <<libs.adoc#lib-directories,libs reference>>.

Reply:
@HoustonPutman , I've applied your perfect wording, slightly fixing reference syntax in according my understanding. Sadly I can't check it quickly locally. I'll check Jenkins refguide build. 
The next question: Do we need to put the same clarification to `/master` ?

Reply:
I made a small change. Also tested it locally, and it looks good. I would port this to master as well.